### ISSUE TITLE
Crashes WS on sendTextMessage

### ISSUE THREAD

**Issue by CheloXL** [adiwajshing/Baileys#48]
If I try to send a message (a simple `client.sendTextMessage("xxxx@c.us", "hello!")`) the library crashes WS on the phone. Of course the message is not received on the target phone, and while the process is running, there is no way to recover from the crash. The app continues crashing until I quit the node process.
By the way, any query (getting all messages, querying for presence, etc) seems to be working fine.

**Comment by CheloXL** [comment]
The problem is with the @c.us. That crashes WS. Replacing that with s.whatsapp.net fixes the problem.
I believe this needs to be documented, or some kind of normalization needs to happen between the library and the user, so we always receive/send just the phone number (without any @xxx) or phone@s.whatsapp.net.

**Comment by adiwajshing** [comment]
It's documented here in the [readme](https://github.com/adiwajshing/Baileys#sending-messages). 

The crash however, is peculiar. Can you send a log?

**Comment by CheloXL** [comment]
Log of what? On node everything is ok. WS crashed on my Android phone (the one that I paired with the library).
I believe that you can test yourself by simply trying to send a message with @c.us.

**Comment by adiwajshing** [comment]
On node, I simply get a 400 response with ‘@c.us’ It does not crash for me. 

I’m unsure what’s happening on an Android device, so it would be great if you could send the log of the app running on Android before it crashes.

**Comment by CheloXL** [comment]
Mmm... I'm on Windows/Node 12.6
Using this code to test

```javascript
client.connect(authJSON)
.then (([user, chats, contacts, unread]) => {
	console.log ("oh hello " + user.name + " (" + user.id + ")")
	console.log ("you have " + unread.length + " unread messages")
	console.log ("you have " + chats.length + " chats")
})
.catch (err => console.log("unexpected error: " + err))
.then(()=>{
	client.sendTextMessage("XXXXXX@c.us", "Hello!").then((a) => console.log(a))
}).catch (err => console.log("unexpected error: " + err));
```
The promise after sendTextMessage never completes (logging incoming messages with a simple console.log on the onMessageRecieved method also shows me that I never get back a response for the message tag/id).

On my phone, I get a system notification that WS crashed. I don't think there are any logs I have access to (don't have my phone rooted).

**Comment by adiwajshing** [comment]
Could you please let me know if the problem is resolved on the latest version (2.0.0) ?

**Comment by adiwajshing** [comment]
Since #55 seems to be the same issue, closing this.
